org.ekstep.questionset-1.0/editor/plugin.js

```diff
@@ -75,8 +75,24 @@ org.ekstep.questionset.EditorPlugin = org.ekstep.contenteditor.basePlugin.extend
     ];
     instance.pluginsRespHandler(pluginsData);
   },
+  checkContentType : function(instance)
+  {
+    let contentMeta = ecEditor.getService(ServiceConstants.CONTENT_SERVICE).getContentMeta(ecEditor.getContext('contentId'));
+    if(contentMeta) {
+      let contentType = contentMeta.contentType;
+      if(contentType === 'SelfAssess') {
+        for( var i = 0; i < instance.manifest.editor.configManifest.length; i++) {
+          if ( instance.manifest.editor.configManifest[i].title === 'Display') {
+          instance.manifest.editor.configManifest.splice(i, 1);
+         }
+        }
+      }
+    }
+    return instance;
```

Comment:
Remove for loop and 
`return _.remove(ab, function(o){ return o.title != 'Display'})`